CHANGELOG.rst

```diff
@@ -0,0 +1,19 @@
+==========
+Changelog
+==========
+
+This project adheres to `Semantic Versioning <https://semver.org/>`_.
+
+
+1.2.0-SNAPSHOT (2021-02-08)
```

Comment:
Should the Changelog be updated here or do you want to do it in a seperate Branch?

Reply:
I would suggest doing that in a different branch. This PR should focus on integrating the TEMPLATE branch into the project.